### PR TITLE
bounty only after merge

### PR DESCRIPTION
**Pull request by emergent-reasons**

### CODE REVIEW

**Comment by emergent-reasons** [comment]
@merc1er Is there anything else we need to do for this re: localization or anything?

**Comment by merc1er** [comment]
You need choose the branch `translations` as the base for this PR (master would work too, but submitting to translations makes it so that Crowdin is updated without the site being published).

Once it is merged in `translations`, you can go ahead and translate the new strings in Crowdin, then you can merge translations into `master`.

File: app/index.html [hunk 1]
```diff
@@ -34,6 +34,10 @@ <h3 class="hero-title">
           <a class="col-11 col-md-5 btn btn-outline-primary btn-lg shadow ml-md-3 mt-3 mt-md-0" href="/$( _lang_ )$/how-to-pledge">$( index.contribute )$</a>
         </div>
 
+        <div class="text-center mt-4">
+          <a href="/$( _lang_ )$/contribute-code" class="small">Contribute code and earn Bitcoin Cash<i data-feather="chevron-right" class="mb-1 ml-1"></i></a>
```
Comment (merc1er) [inline app/index.html:38]: This needs to be in i18n too. My bad, I'll open a PR fixing this later on.

**Review by merc1er — APPROVED** [review]

File: app/index.html (hunk 1 above)
Comment (emergent-reasons) [reply app/index.html:38]: Ok. It's merged now. Can I help with the l18n?

Comment (merc1er) [reply app/index.html:38]: Sorry for the delay on that. I pushed the changes to master

**Pull request merged**